core/types/transaction.go

```diff
@@ -47,8 +47,6 @@ const (
 	AccessListTxType = 0x01
 	DynamicFeeTxType = 0x02
 	BlobTxType       = 0x03
-	// CeloDynamicFeeTxType = 0x7c  old Celo tx type with gateway fee
-	CeloDynamicFeeTxType = 0x7b
```

Comment:
We're losing the "once place to see all tx types", but I think that's ok.

Reply:
It's a tradeoff, but yeah, I think it's ok

Reply:
Actually, I just realised there wasn't ever one place to see all tx types. Optimism decided to put their deposit tx type elsewhere as well, probably for similar reasons.

https://github.com/celo-org/op-geth/blob/2bffde9862720fb090261d28e485f9e9bacf67c5/core/types/deposit_tx.go#L27